servant-examples/getting-started/getting-started.hs

```diff
@@ -0,0 +1,39 @@
+import Network.Wai
+import Network.Wai.Handler.Warp
+import System.Environment
+
+import qualified GS1
+import qualified GS2
+import qualified GS3
+import qualified GS4
+import qualified GS5
+import qualified GS6
+import qualified GS7
+import qualified GS9
+import qualified GS10
+
+app :: String -> (Application -> IO ()) -> IO ()
+app n f = case n of
+  "1" -> f GS1.app
+  "2" -> f GS2.app
+  "3" -> f GS3.app
+  "4" -> f GS4.app
+  "5" -> f GS5.app
+  "6" -> f GS6.app
+  "7" -> f GS7.app
+  "8" -> f GS3.app
```

Comment:
no that's on purpose, because "8" is about *servant-client*, running against the webservice from the 3rd example.